.github/workflows/deploy-stack.yml

```diff
@@ -32,10 +32,9 @@ on:
 # are canceled when we do a deploy so they don't fail erroneously.
 concurrency:
   group: e2e-${{ inputs.environment }}
-  cancel-in-progress: false
```

Comment:
I don't know that we want this to be true as it will also cancel in-progress deploys, which may not be safe if we're in the middle of a migration or something like that.